packages/app/src/debug/DebugResults.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="flex gap-8px items-center"
+    data-cy="debug-results-holder"
+  >
+    <ResultCounts
+      :total-passed="results.totalPassed"
+      :total-failed="results.totalFailed"
+      :total-skipped="results.totalSkipped"
+      :total-pending="results.totalPending"
+      :order="['PASSED', 'FAILED', 'SKIPPED', 'PENDING']"
+    >
+      <template #prefix>
+        <div
+          class="text-red-400 font-semibold text-sm flex items-center border-r-1px px-2"
+        >
+          <component
```

Comment:
Why not just

```vue
  <FailedSolidIcon
    class=...
    data-cy=...
/>
```

`<component>` is good for dynamic content - but this isn't. (Could be missing something).

Reply:
This will not always be a `FailedSolidIcon`. In future iterations, this could be a `PassedIcon` or hold other statuses hence the `<component>`